src/App.tsx

```diff
@@ -61,40 +58,10 @@ export default function App() {
         />
 
       </CategoryCarousel>
-      <div>
-        <VisitingCard
-          bigPrice="R$ 250.000"
-          image={imgVisiting}
-          locationText="Condomínio"
-          iptuText='IPTU'
-          price='R$1450,00'
-          condominiumPrice='R$450,00'
-          description=' Jonathan Marques'
-          captions={['Imobilar corretora imobiliária', 'CRECI-SP J22.004']}
-          icon
-        />
 
-        <VisitingCard
-          bigPrice="R$ 2900.00"
-          image={imgVisiting}
-          locationText="Condomínio"
-          iptuText="IPTU"
-          price="R$ 100,00"
-          condominiumPrice="R$ 450,00"
-          description="Jonathan Marques"
-          captions={['Imobilar corretora imobiliária', 'CRECI-SP J22.004']}
-          icon
-          iconColor='red'
-          serviceFeeLabel="Taxas Extras"
-          serviceFeeValue="R$ 100,00"
-          rentLabel="Aluguel Mensal"
-          rentValue="R$ 2.250,00"
-          value='Total'
-          amount='2900,00'
-        />
+      <Banner />
```

Comment:
nao tem um separator entre o banner e o carousel e outro entre o banner e o footer?

```suggestion
      <Separator size="large" />
      <Banner />
      <Separator size="large" />
```

Reply:
eu ainda nao tinha definido esse espaçamento